docs/en/getting_started.md

```diff
@@ -144,7 +74,7 @@ pip install -r requirements/build.txt
 pip install -v -e .  # or "python setup.py develop"
 ```
 
-**e. Install [MMSegmentation](https://github.com/open-mmlab/mmsegmentation).**
+**Step 2.** Install [MMSegmentation](https://github.com/open-mmlab/mmsegmentation).
```

Comment:
Change v2.19.0 in line 142 to v2.24.0 because the required mmdet has been updated